src/Spice86.Core/Emulator/CPU/CfgCpu/ParsedInstruction/Instructions/Cpuid.cs

```diff
@@ -0,0 +1,22 @@
+namespace Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions;
+
+using Spice86.Core.Emulator.CPU.CfgCpu.Ast.Builder;
+using Spice86.Core.Emulator.CPU.CfgCpu.Ast.Instruction;
+using Spice86.Core.Emulator.CPU.CfgCpu.InstructionExecutor;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Prefix;
+using Spice86.Core.Emulator.CPU.Exceptions;
+using Spice86.Shared.Emulator.Memory;
+using Spice86.Shared.Utils;
+
+public class Cpuid(SegmentedAddress address, InstructionField<ushort> opcodeField, List<InstructionPrefix> prefixes)
+    : CfgInstruction(address, opcodeField, prefixes, 1) {
```

Comment:
Why not just throwing invalid opcode exception in the parser?
If you really want to do it here, calling HandleCpuException here is not correct I think, it's better to rely on the centralized cfgcpu handler.

Reply:
Mind to propose a proper implementation of this case, as discussed in Discord? (with CPUID being used as a probe for the CPU type)